src/main/kotlin/cache/loaders/RegionLoader.kt

```diff
@@ -51,13 +53,14 @@ class RegionLoader(
         }
 
         val inputStream = ByteBuffer.wrap(map)
+        val revision = parseCacheRevision(cacheLibrary.path)
 
         val tiles = Array(Z) {
             Array(X) {
                 Array(Y) {
                     val tile = RegionDefinition.Tile()
                     while (true) {
-                        val attribute: Int = inputStream.readUnsignedByte()
+                        val attribute: Int = if (revision >= 209) inputStream.readUnsignedShort() else inputStream.readUnsignedByte()
```

Comment:
`revision >= 209` might be better extracted as a variable or field (e.g. `private val largeOverlayIds get() = cacheRevision >= 209`) such that it's clearer that it's the same condition in both places, and so that the "magic number" 209 and the meaning of the if-statements become clearer in context.

Reply:
Done